roles/infra/tasks/main.yml

```diff
@@ -8,17 +8,51 @@
       }
     dest: "{{ terraform_project_path }}/backend.tf"
 
+- name: Create Terraform plan
+  community.general.terraform:
+    binary_path: "{{ terraform_binary_path or omit }}"
+    project_path: "{{ terraform_project_path }}"
```

Comment:
currently no default, but think I noticed an oddity yesterday; seemed it couldn't be e.g. `tf_project`, it had to be `./tf_project`. I didn't investigate futher as I was skunkworks-ing.